app/components/listing/feedback/FeedbackModal.jsx

```diff
@@ -39,19 +40,22 @@ const FeedbackModal = ({ service, resource, closeModal }) => {
     setReview(e.target.value);
   };
 
+  const isDownVote = vote === DOWNVOTE;
+
   const handleNextStep = () => (
-    (vote === DOWNVOTE && step === 'start') ? setStep('tags') : setStep('review')
+    (isDownVote && step === 'start') ? setStep('tags') : setStep('review')
   );
 
   const handlePrevStep = () => (
-    (vote === DOWNVOTE && step === 'review') ? setStep('tags') : setStep('start')
+    (isDownVote && step === 'review') ? setStep('tags') : setStep('start')
   );
 
   const handleSubmit = () => {
     setIsSubmitted('submitting');
-    const tags = tagOptions
-      .filter(({ selected }) => selected)
-      .map(({ tag }) => tag);
+
+    const tags = isDownVote
+      ? tagOptions.filter(({ selected }) => selected).map(({ tag }) => tag)
+      : [];
```

Comment:
We wanna keep `review` and `tags` state consistent if user toggles between upvote and downvote but need to make sure we send empty tags before submit since tags associated with downvote only.